docs/adr/013-add-provisioning-saga.md

```diff
@@ -0,0 +1,72 @@
+# 13. Add Provisioning Saga Workflow
+
+Authors: Adi Abramovich
+
+## Status
+
+Proposed
+
+## Problem Statement
+
+This ADR exists because there is a need of making the build and launch workflow more user-friendly. 
+this ADR will include three propositions regarding how should we implement this feature and what should we take into consideration.
+
+## Goals
+
+* Make the saga workflow fast, secure, and simple
+* Choose one of the architectures prop3os3ed
```

Comment:
encoding error? :thinking: xD

I'd maybe go with `Decide on the solution architecture`

Reply:
Threre is still `prop3os3ed` which is I believe not a word :-)